<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>WebEidDialog</class>
 <widget class="QDialog" name="WebEidDialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>520</width>
    <height>460</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string notr="true">Web eID</string>
  </property>
  <property name="styleSheet">
   <string notr="true">QWidget {
background-color: white;
font-size: 14px;
}
QPushButton {
font-size: 17px;
border: 0px;
border-radius: 3px;
height: 45px;
padding-left: 19px;
padding-right: 19px;
min-width: 30px;
background-color: #EFEFEF;
}
QPushButton::hover, QPushButton::focus {
background-color: #DEDEDE;
}
#page1TitleLabel , #page2TitleLabel, #page3TitleLabel, #page4TitleLabel {
color: #003168;
font-size: 32px;
font-weight: bold;
}
#smartCardErrorLabel {
border: 1px solid rgba(240,78,35,0.1);
border-radius: 3px;
padding: 10px;
color: #F04E23;
}
#selectCertificateInfoLabel, #authenticateCertificateInfoLabel, #signCertificateInfoLabel {
border: 1px solid rgba(0,49,104,0.1);
border-radius: 35px;
min-height: 70px;
padding: 0px 20px 0px 28px;
}
#authenticationPinInput, #signingPinInput {
border: 1px solid #113F8E;
border-radius: 3px;
min-width: 165px;
min-height: 45px;
max-width: 165px;
max-height: 45px;
font-size: 20px;
font-weight: bold;
}
#authenticationPinErrorLabel, #signingPinErrorLabel {
color: #F04E23;
}
#selectCertificateOriginLabel, #authenticateOriginLabel, #signOriginLabel {
color: #003168;
font-weight: bold;
}</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout_2">
   <property name="leftMargin">
    <number>20</number>
   </property>
   <property name="topMargin">
    <number>20</number>
   </property>
   <property name="rightMargin">
    <number>20</number>
   </property>
   <property name="bottomMargin">
    <number>20</number>
   </property>
   <item>
    <widget class="QStackedWidget" name="pageStack">
     <property name="currentIndex">
      <number>0</number>
     </property>
     <widget class="QWidget" name="selectCardPage">
      <layout class="QVBoxLayout" name="selectCardPageLayout">
       <property name="spacing">
        <number>10</number>
       </property>
       <property name="leftMargin">
        <number>0</number>
       </property>
       <property name="topMargin">
        <number>0</number>
       </property>
       <property name="rightMargin">
        <number>0</number>
       </property>
       <property name="bottomMargin">
        <number>0</number>
       </property>
       <item>
        <widget class="QLabel" name="page1TitleLabel">
         <property name="focusPolicy">
          <enum>Qt::TabFocus</enum>
         </property>
         <property name="text">
          <string>Smart Card</string>
         </property>
        </widget>
       </item>
       <item>
        <layout class="QHBoxLayout" name="page1HorizontalLayout1">
         <item alignment="Qt::AlignLeft">
          <widget class="QLabel" name="selectCardDescriptionLabel">
           <property name="focusPolicy">
            <enum>Qt::TabFocus</enum>
           </property>
           <property name="text">
            <string>Select a smart card device</string>
           </property>
          </widget>
         </item>
         <item alignment="Qt::AlignLeft">
          <widget class="QLabel" name="spinnerLabel">
           <property name="text">
            <string notr="true">...</string>
           </property>
          </widget>
         </item>
         <item>
          <spacer name="page1HorizontalSpacer1">
           <property name="orientation">
            <enum>Qt::Horizontal</enum>
           </property>
           <property name="sizeHint" stdset="0">
            <size>
             <width>40</width>
             <height>20</height>
            </size>
           </property>
          </spacer>
         </item>
        </layout>
       </item>
       <item>
        <layout class="QHBoxLayout" name="page1HorizontalLayout2" stretch="0,1">
         <property name="spacing">
          <number>20</number>
         </property>
         <property name="topMargin">
          <number>15</number>
         </property>
         <item>
          <widget class="QLabel" name="cardChipIcon">
           <property name="pixmap">
            <pixmap resource="web-eid-resources.qrc">:/smartcard-chip.png</pixmap>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QLabel" name="connectCardLabel">
           <property name="focusPolicy">
            <enum>Qt::TabFocus</enum>
           </property>
           <property name="text">
            <string>Insert a smart card</string>
           </property>
          </widget>
         </item>
        </layout>
       </item>
       <item>
        <widget class="QLabel" name="smartCardErrorLabel">
         <property name="focusPolicy">
          <enum>Qt::TabFocus</enum>
         </property>
         <property name="text">
          <string>Insert a smart card</string>
         </property>
         <property name="wordWrap">
          <bool>true</bool>
         </property>
        </widget>
       </item>
       <item>
        <spacer name="page1VerticalSpacer">
         <property name="orientation">
          <enum>Qt::Vertical</enum>
         </property>
         <property name="sizeHint" stdset="0">
          <size>
           <width>20</width>
           <height>40</height>
          </size>
         </property>
        </spacer>
       </item>
      </layout>
     </widget>
     <widget class="QWidget" name="selectCertificatePage">
      <layout class="QVBoxLayout" name="selectCertificatePageLayout">
       <property name="spacing">
        <number>10</number>
       </property>
       <property name="leftMargin">
        <number>0</number>
       </property>
       <property name="topMargin">
        <number>0</number>
       </property>
       <property name="rightMargin">
        <number>0</number>
       </property>
       <property name="bottomMargin">
        <number>0</number>
       </property>
       <item>
        <widget class="QLabel" name="page2TitleLabel">
         <property name="focusPolicy">
          <enum>Qt::TabFocus</enum>
         </property>
         <property name="text">
          <string>Select Certificate</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLabel" name="selectCertificateDescriptionLabel">
         <property name="focusPolicy">
          <enum>Qt::TabFocus</enum>
         </property>
         <property name="text">
          <string>By choosing the certificate, I agree to submit my name and personal identification number to the website</string>
         </property>
         <property name="wordWrap">
          <bool>true</bool>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLabel" name="selectCertificateOriginLabel">
         <property name="focusPolicy">
          <enum>Qt::TabFocus</enum>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLabel" name="selectCertificateInfoLabel">
         <property name="focusPolicy">
          <enum>Qt::TabFocus</enum>
         </property>
         <property name="text">
          <string notr="true">Authentication:
Issuer:
Valid:</string>
         </property>
         <property name="wordWrap">
          <bool>true</bool>
         </property>
        </widget>
       </item>
       <item>
        <spacer name="page2VerticalSpacer">
         <property name="orientation">
          <enum>Qt::Vertical</enum>
         </property>
         <property name="sizeHint" stdset="0">
          <size>
           <width>20</width>
           <height>40</height>
          </size>
         </property>
        </spacer>
       </item>
      </layout>
     </widget>
     <widget class="QWidget" name="authenticatePage">
      <layout class="QVBoxLayout" name="authenticatePageLayout">
       <property name="spacing">
        <number>10</number>
       </property>
       <property name="leftMargin">
        <number>0</number>
       </property>
       <property name="topMargin">
        <number>0</number>
       </property>
       <property name="rightMargin">
        <number>0</number>
       </property>
       <property name="bottomMargin">
        <number>0</number>
       </property>
       <item>
        <widget class="QLabel" name="page3TitleLabel">
         <property name="focusPolicy">
          <enum>Qt::TabFocus</enum>
         </property>
         <property name="text">
          <string>Authenticate</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLabel" name="authenticateDescriptionLabel">
         <property name="focusPolicy">
          <enum>Qt::TabFocus</enum>
         </property>
         <property name="text">
          <string>By confirming authentication, I agree to submit my name and personal identification number to the website</string>
         </property>
         <property name="wordWrap">
          <bool>true</bool>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLabel" name="authenticateOriginLabel">
         <property name="focusPolicy">
          <enum>Qt::TabFocus</enum>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLabel" name="authenticateCertificateInfoLabel">
         <property name="focusPolicy">
          <enum>Qt::TabFocus</enum>
         </property>
         <property name="text">
          <string notr="true">Authentication:
Issuer:
Valid:</string>
         </property>
         <property name="wordWrap">
          <bool>true</bool>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLabel" name="authenticationPinTitleLabel">
         <property name="font">
          <font>
           <pointsize>-1</pointsize>
           <weight>75</weight>
           <bold>true</bold>
          </font>
         </property>
         <property name="text">
          <string>Please enter authentication PIN (PIN 1):</string>
         </property>
         <property name="alignment">
          <set>Qt::AlignCenter</set>
         </property>
         <property name="buddy">
          <cstring>authenticationPinInput</cstring>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QProgressBar" name="authenticatePinEntryTimeoutProgressBar">
         <property name="textVisible">
          <bool>false</bool>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLineEdit" name="authenticationPinInput">
         <property name="maxLength">
          <number>12</number>
         </property>
         <property name="echoMode">
          <enum>QLineEdit::Password</enum>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLabel" name="authenticationPinErrorLabel">
         <property name="focusPolicy">
          <enum>Qt::TabFocus</enum>
         </property>
         <property name="alignment">
          <set>Qt::AlignCenter</set>
         </property>
        </widget>
       </item>
       <item>
        <spacer name="page3VerticalSpacer">
         <property name="orientation">
          <enum>Qt::Vertical</enum>
         </property>
         <property name="sizeHint" stdset="0">
          <size>
           <width>20</width>
           <height>40</height>
          </size>
         </property>
        </spacer>
       </item>
      </layout>
     </widget>
     <widget class="QWidget" name="signPage">
      <layout class="QVBoxLayout" name="signPageLayout">
       <property name="spacing">
        <number>10</number>
       </property>
       <property name="leftMargin">
        <number>0</number>
       </property>
       <property name="topMargin">
        <number>0</number>
       </property>
       <property name="rightMargin">
        <number>0</number>
       </property>
       <property name="bottomMargin">
        <number>0</number>
       </property>
       <item>
        <widget class="QLabel" name="page4TitleLabel">
         <property name="focusPolicy">
          <enum>Qt::TabFocus</enum>
         </property>
         <property name="text">
          <string>Sign</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLabel" name="signDescriptionLabel">
         <property name="focusPolicy">
          <enum>Qt::TabFocus</enum>
         </property>
         <property name="text">
          <string>By confirming signing, I agree to submit my name and personal identification number to the website</string>
         </property>
         <property name="wordWrap">
          <bool>true</bool>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLabel" name="signOriginLabel">
         <property name="focusPolicy">
          <enum>Qt::TabFocus</enum>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLabel" name="signCertificateInfoLabel">
         <property name="focusPolicy">
          <enum>Qt::TabFocus</enum>
         </property>
         <property name="text">
          <string notr="true">Signature:
Issuer:
Valid:</string>
         </property>
         <property name="wordWrap">
          <bool>true</bool>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QToolButton" name="docHashFingerprintToggleButton">
         <property name="styleSheet">
          <string notr="true">border: none</string>
         </property>
         <property name="text">
          <string>Show document fingerprint</string>
         </property>
         <property name="checkable">
          <bool>true</bool>
         </property>
         <property name="toolButtonStyle">
          <enum>Qt::ToolButtonTextBesideIcon</enum>
         </property>
         <property name="arrowType">
          <enum>Qt::RightArrow</enum>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLabel" name="docHashFingerprintLabel">
         <property name="focusPolicy">
          <enum>Qt::TabFocus</enum>
         </property>
         <property name="wordWrap">
          <bool>true</bool>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLabel" name="signingPinTitleLabel">
         <property name="font">
          <font>
           <pointsize>-1</pointsize>
           <weight>75</weight>
           <bold>true</bold>
          </font>
         </property>
         <property name="text">
          <string>Please enter signing PIN (PIN 2):</string>
         </property>
         <property name="alignment">
          <set>Qt::AlignCenter</set>
         </property>
         <property name="buddy">
          <cstring>signingPinInput</cstring>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QProgressBar" name="signingPinEntryTimeoutProgressBar">
         <property name="textVisible">
          <bool>false</bool>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLineEdit" name="signingPinInput">
         <property name="maxLength">
          <number>12</number>
         </property>
         <property name="echoMode">
          <enum>QLineEdit::Password</enum>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLabel" name="signingPinErrorLabel">
         <property name="focusPolicy">
          <enum>Qt::TabFocus</enum>
         </property>
         <property name="alignment">
          <set>Qt::AlignCenter</set>
         </property>
        </widget>
       </item>
       <item>
        <spacer name="page4VerticalSpacer">
         <property name="orientation">
          <enum>Qt::Vertical</enum>
         </property>
         <property name="sizeHint" stdset="0">
          <size>
           <width>20</width>
           <height>40</height>
          </size>
         </property>
        </spacer>
       </item>
      </layout>
     </widget>
    </widget>
   </item>
   <item>
    <widget class="QDialogButtonBox" name="buttonBox">
     <property name="standardButtons">
      <set>QDialogButtonBox::Cancel|QDialogButtonBox::Ok</set>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <resources>
  <include location="web-eid-resources.qrc"/>
 </resources>
 <connections/>
</ui>
